README.md

```diff
@@ -1,2 +1,110 @@
-# Webpack-Todo_List
-In this project, you will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed. This simple web page will be built using webpack and served by a webpack dev server.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# JavaScript - Build with Webpack  [Todo List Exercise].
+
+In this project, you will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server. [Find project specifications here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md)
+      
+
+<div align="center">
+
+[![View Code](https://img.shields.io/badge/View%20-Code-green)](https://github.com/errea/Webpack-Todo_List/tree/feature/webpack-test)
+[![Github Issues](https://img.shields.io/badge/GitHub-Issues-orange)](https://github.com/errea/Webpack-Todo_List/issues)
+[![GitHub Pull Requests](https://img.shields.io/badge/GitHub-Pull%20Requests-blue)](https://github.com/errea/Webpack-Todo_List/pulls)
+
+</div>
+
+## 📝 Contents
+
+<p align="center">
+<a href="#with">Built with</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
+<a href="#gs">Getting started</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
+<a href="#author">Author</a>
+</p>
+
+
+
+># [Description]
+>In this project, you will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server
+
+
+## 🔧 Built with<a name = "with"></a>
+
+- JavaScript
+- DOM
+- [Webpack config](hhttps://webpack.js.org/guides/getting-started/)
+
+![screenshot](./src/img/screenshot-todo.png)
+## Getting Started <a name = "gs"></a>
+> To get a local copy up and running follow these simple example steps.
+
+<details>
+  <summary>Get instructions</summary>
+
+```
+$ cd <folder>
+```
+
+~~~bash
+$ git clone https://github.com/errea/Webpack-Todo_List.git
+$ cd Webpack-Todo_List
+~~~
+
+Run `npm start` opens on `http://localhost:8080/` in your browser.
+
```

Comment:
- [ ] I think you have missed one command before `npm start`, it should be `npm install` to install the node modules and webpacker otherwise it won't work. Please mention this in your README file.